Training/2.0 Libraries.md

```diff
@@ -51,14 +63,25 @@ Normally we would want to be able to extract the information from tables directl
 
 [Training2_example.csv](https://github.com/alepgr/gubmes-hda/blob/main/Training/Training2_example.csv)
 
-```
+``` import pandas as pd
+``` open("training2data.xlsx")
+``` xl = pd.ExcelFile("training2data.xlsx")
+``` xl.sheet_names
+``` df = xl.parse("Sheet1")
+``` print(pd.DataFrame(df))
 
 ```
 
-### Task 4a)
+### Task 4b)
 Using your DataFrame from Task 4a), calculate the average systolic blood pressure of Dementia patients.
 
-```
+``` import pandas as pd
+``` open("training2data.xlsx")
+``` df = pd.read_excel("training2data.xlsx")
+``` dff = pd.DataFrame(df)
+``` dem = dff[dff['Condition'].isin(['Dementia'])]
+``` sys = dem[['Systolic Blood Pressure']]
+``` average = print((sys.mean()))
 
```

Comment:
Correct method for filtering and averaging, but see previous comment for loading a .csv file directly.
Also note, you can not save the output of a print() function to a variable. Print() will only send the output to your terminal.